agent/structs/discovery_chain.go

```diff
@@ -107,6 +107,9 @@ type DiscoveryGraphNode struct {
 
 	// fields for Type==resolver
 	Resolver *DiscoveryResolver `json:",omitempty"`
+
+	// shared by Type==resolver || Type==splitter
+	LoadBalancer *LoadBalancer `json:",omitempty"`
```

Comment:
There's a mirror image of this that needs updating in `api/discovery_chain.go`

Reply:
Fixed